pkg/cmd/pulumi/policy/io.go

```diff
@@ -76,7 +80,50 @@ func InstallPolicyPackDependencies(ctx context.Context, root string, proj *works
 		return fmt.Errorf("failed to load language plugin %s: %w", proj.Runtime.Name(), err)
 	}
 
-	if err = lang.InstallDependencies(plugin.InstallDependenciesRequest{Info: programInfo}); err != nil {
+	stdout, stderr, done, err := lang.InstallDependencies(plugin.InstallDependenciesRequest{Info: programInfo})
+	if err != nil {
+		return fmt.Errorf("installing dependencies failed: %w", err)
+	}
+
+	// We'll use a WaitGroup to wait for the stdout (1) and stderr (2) readers to be fully drained, as well as for the
```

Comment:
Given you wrote this three times is it worth throwing in cmdutils or something?

Reply:
Good shout. Moved to a new `pkg/util/cmdutil`.